js/main.js

```diff
@@ -34,6 +41,92 @@ Site = {
   },
 };
 
+Site.Mailchimp = {
+  $submitButton: Site.$mailchimpForm.find('input[type="submit"]'),
+  $emailInput: Site.$mailchimpForm.find('input[type="email"]'),
+
+  init: function() {
+    var _this = this;
+      
+    _this.$submitButton.on('click', function (event) {
+      event.preventDefault();
+      _this.$submitButton.prop('disabled', true);
+
```

Comment:
should there be some indication of loading/submit. Could also just be styling for disabled submit button